arguments.py

```diff
@@ -28,40 +28,105 @@
 def parse_args():
     """Commandline arguments for CORIO Driver."""
     parser = ArgumentParser()
-    parser.add_argument("-ti", "--test_input", type=str, required=True,
-                        help="Directory path containing test data input yaml files or "
-                             "input yaml file path.")
-    parser.add_argument("-v", "--verbose", action="store_true",
-                        help="log level used verbose(debug), default is info.")
-    parser.add_argument("-us", "--use_ssl", type=lambda x: bool(strtobool(str(x))), default=True,
-                        help="Use HTTPS/SSL connection for S3 endpoint.")
-    parser.add_argument("-sd", "--seed", type=int, help="seed.",
-                        default=random.SystemRandom().randint(1, 9999999))
-    parser.add_argument("-sk", "--secret_key", type=str, nargs="+",
-                        action=SplitArguments, required=True,
-                        help="One or more(space/comma separated) s3 secret Key or keys in order.")
-    parser.add_argument("-ak", "--access_key", type=str, nargs="+",
-                        action=SplitArguments, required=True,
-                        help="One or more(space/comma separated) s3 access Key or keys in order.")
-    parser.add_argument("-ep", "--endpoint", type=str, required=True,
-                        help="fqdn/ip:port of s3 endpoint for io operations without http/https."
-                             "protocol in endpoint is based on use_ssl flag.",
-                        default="s3.seagate.com")
-    parser.add_argument("-nn", "--number_of_nodes", type=int,
-                        help="number of nodes in k8s system", default=1)
-    parser.add_argument("-sb", "--support_bundle", type=lambda x: bool(strtobool(str(x))),
-                        default=False, help="Capture Support bundle.")
-    parser.add_argument("-hc", "--health_check", type=lambda x: bool(strtobool(str(x))),
-                        default=False, help="Health Check.")
-    parser.add_argument("-tp", "--test_plan", type=str, default=None,
-                        help="jira xray test plan id")
-    parser.add_argument("-dm", "--degraded_mode", type=lambda x: bool(strtobool(str(x))),
-                        default=False,
-                        help="Degraded Mode, True/False")
-    parser.add_argument("-mr", "--s3max_retry", type=int, default=0,
-                        help="Max number of retries in case of any type of failure.")
-    parser.add_argument("-sr", "--sequential_run", action="store_true",
-                        help="Run test sequentially from workload.")
+    parser.add_argument(
```

Comment:
Lots of files getting changed for formatting. Can we revert them as this is not required? cc: @VikasKumarC

Reply:
@DhobaleVishal  Let us do this time, we will follow formatting with black and finally with autopep8 then will proceed to avoid any possible conflict.
Ref: https://pypi.org/project/black/ 
https://pypi.org/project/black/#:~:text=Black%20is%20a%20PEP%208%20compliant%20opinionated%20formatter